### PR TITLE
Moving to Jekyll!

### PR DESCRIPTION
**Pull request by namangoel1**
Adding template code, moving all students and mentors to their own JSON files. [Demo](http://namangoel1.github.com/fossasia.github.io)

### CODE REVIEW

**Comment by namangoel1** [comment]
Just moved the styles to SCSS. @RoonyH have a look at the demo, do I merge? :)

File: index.html [hunk 1]
```diff
@@ -134,1009 +134,93 @@ <h1 class="section__title">About</h1>
...
-        </div>-->
     </section>
     <section id="students" class="page-section">
         <div class="wrap">
             <h1 class="section__title">Students</h1>
             <div class="stack stack-clean">
+                {% for student in site.data.students %}
                 <div class="sympl onethird m-half">
-                    <div class="box-team wow bounceInDown" data-wow-delay="0.1s">
+                    <div class="box-team wow tada" data-wow-delay="0.1s">
```
Comment (roonyh) [inline index.html]: lets keep bounceindown?

**Comment by roonyh** [comment]
@namangoel1 Thanks! I am all for this. Lets get this merged asap. @fossasia/fossasia-gci-students what do you people think about this.

We are trying to move mentors and students' data to a json file and load them using an html template. Github pages works great with [jekyll](https://help.github.com/articles/using-jekyll-with-pages/) so we use it for this purpose. It will be great if you can get yourself familiarized with jekyll since many new tasks  realated to jekyll will come your way if this experiment is a success.

http://jekyllrb.com/

http://jekyllrb.com/docs/datafiles/

File: index.html (hunk 1 above)
Comment (namangoel1) [inline index.html]: Sure yes, must've resulted when I was copy/pasting. I'll edit and push.

**Comment by Ludwant** [comment]
That sounds fine! I will start looking at jekyll now.

**Comment by tradzik** [comment]
I think this is good idea. But we'll need to change README.md file, with instructions about adding image.

**Comment by namangoel1** [comment]
@ty221 [already done](https://github.com/namangoel1/fossasia.github.io/commit/8343900d41c6655afc3eea46fe352d40498877d0). This definitely can be improved, though.

**Comment by PolBaladas** [comment]
Fine! Good Idea, I'll take a look at jekyll

**Comment by roonyh** [comment]
I think the random delay generator could be better, but lets get this in since its a major improvement.

**Comment by namangoel1** [comment]
@RoonyH I agree, but Liquid doesn't have a built-in random function, and wow.js does not allow adding delays through JavaScript. I had an idea to modify wow.js directly, shall I try that?

**Pull request merged**